tests/testthat/test-pkg-arrow.R

```diff
@@ -480,3 +480,18 @@ test_that("arrow translation for sign() works", {
     tibble::tibble(dbl = c(-1, -1, -1, 0, 1))
   )
 })
+
+test_that("arrow translation for if_else() works", {
+  skip_if_not(has_arrow_with_substrait())
+
+  expect_equal(
+    example_data[1:5, "dbl"] %>%
```

Comment:
I quite like having the same dataset to reuse across multiple tests.  I guess it could make sense to leave it exclusively for the `compare_dplyr_bindings()` functions, where you don't need to know what the dataset looks like?